test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -753,5 +754,49 @@ public static string AddSignatureLogPrefix(string log, PackageIdentity package,
         {
             return $"{string.Format(_signatureLogPrefix, package.Id, package.Version, source)} {log}";
         }
+
+        public static HashAlgorithm GetSha1HashProvider()
+        {
+#if !IS_CORECLR
+            if (AllowFipsAlgorithmsOnly())
+            {
+                return new SHA1CryptoServiceProvider();
+            }
+            else
```

Comment:
I don't know that FIPS is important in unit test code.

I think that you could undo all changes to this file and where you have `SigningTestUtility.GetSha1HashProvider()` just call `SHA1.Create()`.  Then undo changes to this file.

Reply:
Thanks. Fixed.